src/register/data/utils.js

```diff
@@ -1,7 +1,12 @@
 import { getConfig, snakeCaseObject } from '@edx/frontend-platform';
 
-import { LETTER_REGEX, NUMBER_REGEX } from '../../data/constants';
+import { LETTER_REGEX, NUMBER_REGEX, VALID_EMAIL_REGEX } from '../../data/constants';
 import messages from '../messages';
+import validateEmail from '../RegistrationFields/EmailField/validator';
+import validateName from '../RegistrationFields/NameField/validator';
+import validateUsername from '../RegistrationFields/UsernameField/validator';
+
+export const emailRegex = new RegExp(VALID_EMAIL_REGEX, 'i');
```

Comment:
this can be removed

Reply:
Hi @syedsajjadkazmii 
Why remove `validateUsername` and `validateName`? Or was this noted by chance?

Reply:
> Hi @syedsajjadkazmii Why remove `validateUsername` and `validateName`? Or was this noted by chance?

I only meant for emailRegex. Github picked those lines as well. :)